homeassistant/components/econet/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]"
+    },
+    "abort": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]"
```

Comment:
Please add an abort string for already configured. Use the common reference.
```suggestion
```

Reply:
Added